src/components/AuthenticatedWrapper.vue

```diff
@@ -91,6 +105,25 @@ export default {
   },
 
   methods: {
+    connectWebsocket() {
+      const index = Vue._installedPlugins.indexOf(VueNativeSock);
+      // Remove the Websocket plugin if it is already installed, so we can re-init and change the uri.
+      if (index > -1) {
+        this.$disconnect();
```

Comment:
is this asynchronous? On desktop I've seen the websocket sometimes seemingly get in a bad state after switching networks occasionally, but hard to reproduce or tell what is going on.

Reply:
No it is not.

Reply:
The newer version of the vue websocket library allows passing the url to connect, so this isn’t necessary anymore. I’ll merge it over later tomorrow when I do merge from desktop.